cumulusci/robotframework/Salesforce.py

```diff
@@ -499,6 +506,111 @@ def salesforce_insert(self, obj_name, **kwargs):
         self.store_session_record(obj_name, res["id"])
         return res["id"]
 
+    def _salesforce_generate_object(self, obj_name, **fields):
+        obj = {"attributes": {"type": obj_name}}  # Object type to create
+        obj.update(fields)
+        return obj
+
+    def generate_test_data(self, obj_name, number_to_create, **fields):
+        """Returns an array of dictionaries with template-formatted arguments appropriate for a Collection Insert.
+            Use ``{{number}}`` to represent the unique index of the row in the list of rows.
```

Comment:
According to PEP 257, in multi-line docstrings the indentation after the first line should match the indentation of the quotes that start the docstring.

Reply:
Fixed.

https://github.com/SFDO-Tooling/CumulusCI/pull/1218/commits/0dad119559cbd7e98387c4eab57b8b66b9c43816#diff-17aaca465159b9f94d1457f75bf63cf5R514